1546 add deb permission note (#1735)
sensu/sensu-docs: content/sensu-go/5.0/guides/systemd-logs.md

{{< highlight shell>}}
logrotate -d /etc/logrotate.d/sensu.conf
{{< /highlight>}}

> _NOTE: For Ubuntu systems, there are some additional permissions required. After you've set up logging to disk on Ubuntu, ensure you run `chown -R syslog:adm /var/log/sensu` so that syslog will be able to write to that directory._

Reviewer: ```suggestion
_NOTE: On Ubuntu systems, be sure to run `chown -R syslog:adm /var/log/sensu` so syslog can write to that directory._
```
